src/components/PeopleTabs.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, useParams } from 'react-router-dom';
+import { findParent } from '../utils/findParent';
+import cn from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTabs: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(p => (
+          <tr
+            data-cy="person"
+            key={p.slug}
+            className={cn({ 'has-background-warning': slug === p.slug })}
+          >
+            <td>
+              <Link
+                to={`../${[p.slug]}`}
```

Comment:
The `to` attribute in the `Link` component should be a string, not an array. Change `to={`../${[p.slug]}`}` to `to={`../${p.slug}`}` to ensure the link is correctly formatted.